src/plugins/ResizeReorder/ResizeReorderCell.js

```diff
@@ -0,0 +1,249 @@
+import ReorderHandle from 'ReorderHandle';
+import joinClasses from 'joinClasses';
+import cx from 'cx';
+
+/**
+ * A plugin that can make use of ResizerKnob and ReorderHandle to provide the
+ * resize and reorder functionality to the columns.
+ *
+ * Pass this to the `header` prop of the `Column` to invoke resizer/reorder handle.
+ * `onColumnResizeEndCallback` and `onColumnReorderEndCallback` activate the respective
+ * functionality. Either one or both can be used together.
+ *
+ */
+class ResizeReorderCell extends React.PureComponent {
+
+  static propTypes = {
+
+    /**
+     * Outer height of the cell.
+     */
+    height: PropTypes.number,
+
+    /**
+     * Outer width of the cell.
+     */
+    width: PropTypes.number,
+
+    /**
+     * Optional prop that if specified on the `Column` will be passed to the
+     * cell. It can be used to uniquely identify which column is the cell is in.
+     */
+    columnKey: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number,
+    ]),
+
+    /**
+     * Optional prop that represents the rows index in the table.
+     * For the 'cell' prop of a Column, this parameter will exist for any
+     * cell in a row with a positive index.
+     *
+     * Below that entry point the user is welcome to consume or
+     * pass the prop through at their discretion.
+     */
+    rowIndex: PropTypes.number,
+
+    /**
+     * The height of the table.
+     */
+    tableHeight: PropTypes.number,
+
+    /**
+     * The left offset in pixels of the cell.
+     * Space between cell's left edge and left edge of table
+     */
+    left: PropTypes.number,
+
+    /**
+     * The left offset in pixels of the cell group.
+     */
+    cellGroupLeft: PropTypes.number,
+
+    /**
+     * Whether touch is enabled or not.
+     */
+    touchEnabled: PropTypes.bool,
+
+    /**
+     * If the component should render for RTL direction
+     */
+    isRTL: PropTypes.bool,
+
+    /**
+     * availableScrollWidth returned from ColumnWidths.
+     */
+    availableScrollWidth: PropTypes.number,
+
+    /**
+     * Function to change the scroll position by interacting
+     * with the store.
+     */
+    scrollToX: PropTypes.func,
+
+    /**
+     * Whether the cells belongs to the fixed group
+     */
+    isFixed: PropTypes.bool,
+
+    /**
+     * The minimum width of the column.
+     */
+    minWidth: PropTypes.number,
+
+    /**
+     * The maximum width of the column.
+     */
+    maxWidth: PropTypes.number,
+  };
+
+  initialState = {
+    isColumnReordering: false,
+    displacement: 0,
+  };
+
+  state = { ...this.initialState };
+
+  /**
+   *
+   * @param {Object} data
+   * @param {boolean} data.isColumnReordering
+   * @param {number} data.displacement
+   */
+  updateReorderingData = (data) => {
+    const columnNotMoved = data.displacement === this.state.displacement
+    if (this.state.isColumnReordering && columnNotMoved) {
+      return;
+    }
+
+    this.setState(data);
```

Comment:
Updates the dispalcement in state to translate the div

Reply:
+1